packages/react/src/primitives/Accordion/types.ts

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { ElementType, PrimitiveProps, BaseViewProps } from '../types/view';
+
+export interface BaseAccordionProps extends BaseViewProps {
+  /**
+   * @description
+   * The value of the item(s) to expand.  Use on uncontrolled component.
+   */
+  defaultValue?: string[];
+
+  /**
+   * @description
+   * The controlled value of the item(s) to expand. Must be used in conjunction with onValueChange.
+   */
+  value?: string[];
+
+  /**
+   * @description
+   * Determines whether one or multiple items can be opened at the same time.
+   * Default: false
+   */
+  allowMultiple?: boolean;
+
+  /**
+   * @description
+   * Determines whether the opened item can always be collapsed even if its the only one.
+   * Default: false
+   */
+  isAlwaysOpen?: boolean;
```

Comment:
Tied to the suggestion above on line 26 - Would this property be more descriptive of its use case as `preventCollapse`?